### PR TITLE
Set up calendar component skeleton

### PR DESCRIPTION
**Pull request by michelledhuang23**

### CODE REVIEW

**Comment by michelledhuang23** [comment]
@sbtorres Please review! Thanks!

File: client/src/calendar.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable class-methods-use-this */
```
Comment (sbtorres) [inline client/src/calendar.jsx]: Did you actually want to commit this with the eslint disabled?

File: db/fake_data.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ].join('-');
+}
+
+for (let i = 0; i < 101; i += 1) {
+  const listing = {
+    price: faker.random.number({ min: 30, max: 1000 }),
+    num_reviews: faker.random.number({ min: 0, max: 300 }),
+    avg_rating: faker.random.number({ min: 0, max: 50 }),
+    service_fee_pct: faker.random.number({ min: 105, max: 110 }),
+    min_nights: faker.random.number({ min: 2, max: 30 }),
```
Comment (sbtorres) [inline db/fake_data.js]:
```suggestion
    min_nights: faker.random.number({ min: 2, max: 6 }),
```
Maybe a maximum of 6 for the minimum number or nights? 30 seems aggressive to me.

File: db/fake_data.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    (mm > 9 ? '' : '0') + mm,
+    (dd > 9 ? '' : '0') + dd,
+  ].join('-');
+}
+
+for (let i = 0; i < 101; i += 1) {
+  const listing = {
+    price: faker.random.number({ min: 30, max: 1000 }),
+    num_reviews: faker.random.number({ min: 0, max: 300 }),
+    avg_rating: faker.random.number({ min: 0, max: 50 }),
```
Comment (sbtorres) [inline db/fake_data.js]:
```suggestion
    avg_rating: faker.random.number({ min: 0, max: 5 }),
```
Did you mean 5? I am guessing this is average number of stars from the variable name

File: db/fake_data.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  const toDate = faker.date.between(formatDate(fromDateTemp), '2019-12-31');
+  const booking = {
+    listing_id: faker.random.number({ min: 1, max: 101 }),
+    from_date: fromDate,
+    to_date: toDate,
+  };
+  availabilityData.push(booking);
+}
+
+db.sync
```
Comment (sbtorres) [inline db/fake_data.js]: This is cool! I'll have to check out sequelize next time I seed a database.

File: db/sequelize.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  // attributes
+  listing_id: {
+    type: Sequelize.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  price: Sequelize.INTEGER,
+  num_reviews: Sequelize.INTEGER,
+  avg_rating: Sequelize.INTEGER,
+  service_fee_pct: Sequelize.INTEGER,
```
Comment (sbtorres) [inline db/sequelize.js]: Maybe a decimal data type for this?

File: client/src/calendar.jsx (hunk 1 above)
Comment (michelledhuang23) [reply client/src/calendar.jsx]: oops...I'll remove. Thanks!

File: db/fake_data.js (hunk 2 above)
Comment (michelledhuang23) [reply db/fake_data.js]: I'm covering short-term rentals as well!

File: db/fake_data.js (hunk 3 above)
Comment (michelledhuang23) [reply db/fake_data.js]: Yea it does seem confusing, I'll look into it and maybe just change it to a float value.

File: db/sequelize.js (hunk 5 above)
Comment (michelledhuang23) [reply db/sequelize.js]: Yep, I will look into it!

File: client/src/calendar.jsx [hunk 6]
```diff
@@ -1,35 +1,95 @@
...
+    const { dateContext } = this.state;
+    return dateContext.daysInMonth();
+  }
+
+  getCurrentDate() {
+    const { dateContext } = this.state;
+    return dateContext.get('date');
+  }
+
+  getCurrentDay() {
```
Comment (sbtorres) [inline client/src/calendar.jsx]: Probably could factor out this object destructuring to make it more DRY

Comment (michelledhuang23) [reply client/src/calendar.jsx]: airbnb style guide enforces object destructuring, so I have to D:

**Pull request merged**